tests/utils/model_configs.py

```diff
@@ -538,6 +538,115 @@ def _update_and_add_testing_config(
     },
 )
 
+_update_and_add_testing_config(
+    # Tests logit distillation.
+    "mistral",
+    "mistral_distill_logits",
+    updates={
+        ("model", "base_model", "head", "distillation_model"): "teacher",
+        ("reference_models"): {
+            "teacher": {
+                "model": {
+                    "base_model": {
+                        "embeddings": {
+                            "vocab_size": MODEL_TEST_VOCAB_SIZE,
+                        },
+                        "decoder": {
+                            "block": {
+                                "mixer": {
+                                    "head_groups": 2,
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+        },
+    },
+    megatron_args=None,
+    checkpoint_format=MistralCheckpointFormat,
+    groups={
+        ModelTestingGroup.basic: ModelTestingGroupAction.normal,
+        ModelTestingGroup.checkpoint: ModelTestingGroupAction.unimportant,
+        ModelTestingGroup.convert: ModelTestingGroupAction.unimportant,
+        ModelTestingGroup.generate: ModelTestingGroupAction.unimportant,
+        ModelTestingGroup.megatron: ModelTestingGroupAction.not_implemented,
+        ModelTestingGroup.distributed: ModelTestingGroupAction.broken,  # failing: tp2, stp2, stp2_ce4
```

Comment:
We'll probably want to leave these as unimportant and run once in a while, because the testing suite can't really support many distributed runs.

Reply:
Makes sense. Some of the distributed tests are failing currently, so let's leave it as `broken` for now?